packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -100,28 +117,42 @@ export const TitleAndDatePickerComponent = ({
 
   return (
     <Wrapper>
-      <Content>
-        <FlexSpaceBetween>
-          <Pin isPinned={pinnedOverlay === mapOverlayCode} onChange={handlePinChange} />
-          <ContentText>{isMeasureLoading ? <CircularProgress size={22} /> : name}</ContentText>
-        </FlexSpaceBetween>
-        <Switch checked={isSwitchedOn} onChange={handleSwitchChange} />
-      </Content>
-
-      {showDatePicker && (
-        <MeasureDatePicker>
-          <DateRangePicker
-            key={name} // force re-create the component on measure change, which resets initial dates
-            granularity={periodGranularity}
-            startDate={startDate}
-            endDate={endDate}
-            min={datePickerLimits.startDate}
-            max={datePickerLimits.endDate}
-            onSetDates={updateMeasurePeriod}
-            isLoading={isMeasureLoading}
-          />
-        </MeasureDatePicker>
+      {isMeasureLoading ? (
+        <SkeletonHeader mt="10px">
```

Comment:
The loading section appears to be a different height to the content section and so the side bar moves up and down when it starts and stops loading. I think adding 10px margin bottom will fix it.

Reply:
Sounds good, these component have their own padding and it costs me some times to fix it. Is it ok to use 'min-height' to make them consistent?

Reply:
yep no problem